src/pages/list/Todo/Todo.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { CiEdit, CiSquareRemove } from "react-icons/ci";
+import EditTodoForm from "./EditTodoForm/EditTodoForm";
+import { client } from "api/client";
+
+const Todo = ({ id, title, deleteTodo, content, todoList, setTodoList }) => {
+  const [isEdit, setIsEdit] = useState(false);
+  const [editedTodo, setEditedTodo] = useState({ id: "", title: "", content: "" });
+  const navigate = useNavigate();
+
+  const getEditedTodo = (e) => {
+    const { name, value } = e.target;
+    setEditedTodo({ ...editedTodo, [name]: value });
+  };
+
+  const toggleEdit = (e, selectedId) => {
+    setEditedTodo({ ...editedTodo, id: selectedId });
+    setIsEdit((prev) => !prev);
+  };
+
+  const selectTodo = (e) => {
+    navigate(`/${id}`);
+  };
+
+  const closeModal = () => {
+    setIsEdit((prev) => !prev);
+  };
+
+  // const updateElement = (updatedTodo) => {
+  //   const editedIndex = todoList.findIndex(({ id }) => id === updatedTodo.id);
+  //   if (editedIndex !== -1) {
+  //     const updatedTodoList = [...todoList];
+  //     updatedTodoList[editedIndex] = updatedTodo;
+  //     setTodoList(updatedTodoList);
+  //   }
+  // };
+
+  const updateElement = (updatedTodo) => {
+    setTodoList((prevTodoList) => {
+      const editedIndex = prevTodoList.findIndex(({ id }) => id === updatedTodo.id);
+      if (editedIndex !== -1) {
+        const updatedTodoList = [...prevTodoList];
+        updatedTodoList[editedIndex] = updatedTodo;
```

Comment:
```suggestion
    setTodoList((prevTodoList) => {
      const isTodoEdited = prevTodoList.find(({ id }) => id === updatedTodo.id);
      if (isTodoEdited) {
        const updatedTodoList = [...prevTodoList];
        updatedTodoList[editedIndex] = updatedTodo;
```
`-1` 자체로는 어떠한 의미를 표현하지 못하기 때문에(매직 리터럴) 한번 더 생각해야 하는 반면, 그냥 find를 쓰면 되지 않나 싶기도 하고.. 그리고 `-1`이 `아닌 것`은? 이라는 로직은 뭔가 묘하게 이중부정 같기도 하네요 그냥 `찾으면` 이라는 로직이 자연스러워 보입니다

Reply:
find로 수정했습니다!
index찾아서 갈아껴야지~ 생각했는데 
find가 바로 해당 요소를 반환하니까 보기 편하네요..  감사합니다👍🏻